components/eamxx/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -168,12 +177,17 @@ yakl_process_target(scream_rrtmgp_yakl)
 # NOTE: cannot use 'PUBLIC' in target_link_libraries,
 #       since yakl_process_target already used it
 #       with the "plain" signature
+
+#<<<<<<< HEAD
+#find_library(NETCDF_C netcdf HINTS $ENV{NETCDF_C_PATH}/lib)
+#target_link_libraries(scream_rrtmgp_yakl ${NETCDF_C} rrtmgp scream_share)
```

Comment:
Can you explain this change? The second line I'm guessing is due to Kokkos, but what about the NETCDF_C line?

Reply:
i need to ask Jim about this -- iirc i had to set io libs in general, but also some cmake variables for io for rrtmgp separately. so i left this file messy, because i still need to sort it out.